frontend/tests/Factories/Course.php

```diff
@@ -12,7 +12,8 @@ public static function createCourse(
         string $admissionMode = \OmegaUp\Controllers\Course::ADMISSION_MODE_PRIVATE,
         string $requestsUserInformation = 'no',
         string $showScoreboard = 'false',
-        ?int $courseDuration = 120
+        ?int $courseDuration = 120,
+        string $courseAlias = null
```

Comment:
```suggestion
        ?string $courseAlias = null
```

si se le está pasando `null` como valor default, entonces hay que decirle a Psalm que esto puede ser nulo. igual en todos los demás lugares con este patrón.